pennylane/tape/jacobian_tape.py

```diff
@@ -93,6 +93,11 @@ def __init__(self, name=None, do_queue=True):
         self.jacobian_options = {}
         self.hessian_options = {}
 
+    def copy(self, copy_operation=False, tape_cls=None):
+        copied_tape = super().copy(copy_operation=copy_operation, tape_cls=tape_cls)
```

Comment:
```suggestion
    def copy(self, copy_operations=False, tape_cls=None):
        copied_tape = super().copy(copy_operations=copy_operation, tape_cls=tape_cls)
```

@albi3ro apologize for hijacking, just really want to see why the CI is failing!